ros2launch/ros2launch/api/api.py

```diff
@@ -74,19 +67,19 @@ def get_share_file_path_from_package(*, package_name, file_name):
     return matching_file_paths[0]
 
 
-def get_python_launch_file_paths(*, path):
-    """Return a list of paths to Python launch files within a given path."""
-    python_launch_file_paths = []
+def get_launch_file_paths(*, path):
+    """Return a list of paths to launch files within a given path."""
+    launch_file_paths = []
     for root, dirs, files in os.walk(path):
         for file_name in files:
-            if file_name.endswith('launch.py'):
-                python_launch_file_paths.append(os.path.join(root, file_name))
-    return python_launch_file_paths
+            if file_name.endswith(('.launch.py', '.launch.xml', '.launch.yaml')):
```

Comment:
Yeah, adding the `.` to the `launch.py` isn't right. That was intentional so that anything ending with `launch.py` (including `foo_launch.py` and `foo.launch.py`) is acceptable. The same should (in my opinion) be applied to the new xml and yaml ones.

Reply:
I agree.
Also, there's nothing stopping the user to run a file not following these naming conventions.
For example:
```bash
ros2 launch package filename_without_correct_ending
```

Will launch the file (regardless if it is an xml, yaml or python file).
But the tool will not autocomplete the name.